UtilityMemoryBag.lua

```diff
@@ -1,7 +1,127 @@
 -- Utility memory bag by Directsun
--- Version 2.6.0
+-- Version 2.7.0
 -- Fork of Memory Bag 2.0 by MrStump
 
+CONFIG = {
+    MEMORY_GROUP = {
+        -- Use an identical value across bags in the group.
+        -- Set this to 'nil' in order to not have the bag in a group.
+        NAME = nil,

```

Comment:
If you made this a default "nongroup" or something, we could track all Util memory bag guids